From 904e3ea057b28f80235222fd8d4a3afd5eb7c908 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 21 Feb 2017 22:33:18 +0100
Subject: [PATCH] iplink: add missing link type

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1374493
Upstream Status: iproute2.git commit d8cf93de04d93
Conflicts: Patch manually applied - RHEL doesn't support ipvlan but
           macsec support has already been backported.

commit d8cf93de04d9372f478fc2e59e2fe8bce1953b7a
Author: Zhang Shengju <zhangshengju@cmss.chinamobile.com>
Date:   Wed Aug 12 06:03:22 2015 +0000

    iplink: add missing link type

    Add missing link type "bridge_slave".

    Signed-off-by: Zhang Shengju <zhangshengju@cmss.chinamobile.com>
---
 ip/iplink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ip/iplink.c b/ip/iplink.c
index e0e0fe9..f75193f 100644
--- a/ip/iplink.c
+++ b/ip/iplink.c
@@ -97,7 +97,7 @@ void iplink_usage(void)
 		fprintf(stderr, "TYPE := { vlan | veth | vcan | dummy | ifb | macvlan | macvtap |\n");
 		fprintf(stderr, "          bridge | bond | ipoib | ip6tnl | ipip | sit | vxlan |\n");
 		fprintf(stderr, "          gre | gretap | ip6gre | ip6gretap | vti | nlmon |\n");
-		fprintf(stderr, "          bond_slave | geneve | macsec }\n");
+		fprintf(stderr, "          bond_slave | geneve | bridge_slave | macsec }\n");
 	}
 	exit(-1);
 }
-- 
1.8.3.1

